### ISSUE TITLE
ChannelDescriptionChangedEvent

### ISSUE THREAD

**Issue by spacemanspiff2007** [spacemanspiff2007/HABApp#236]
```
ERROR |      _in_dict = {'topic': 'openhab/channels/squeezebox:squeezeboxplayer:myServer:MyDevice:playFavorite/descriptionchanged',
ERROR |                  'payload': '{"field":"STATE_OPTIONS","channelUID":"squeezebox:squeezeboxplayer:myServer:MyDevice:playFavorite","linkedItemNames":["MyDevice_Favorite"],"value":"{\\"options\\":[{\\"value\\":\\"0\\",\\"label\\":\\"MyFavRadioStation\\"}]}","oldValue":"{\\"options\\":[]}"}',
ERROR |                  'type': 'ChannelDescriptionChangedEvent'}
```

**Comment by die-bauerei** [comment]
Might be that I have somewhat similar issue:
2022-04-08 11:54:08.882 [ERROR] [HABApp                              ] - Error Unknown Event: ChannelDescriptionChangedEvent for {'topic': 'openhab/channels/avmfritz:fritzbox:192_168_122_1:apply_template/descriptionchanged', 'payload': '{"field":"COMMAND_OPTIONS","channelUID":"avmfritz:fritzbox:192_168_122_1:apply_template","linkedItemNames":["ApplyTemplate"],"value":"{\\"options\\":[{\\"command\\":\\"tmp6FCB3D-3D3532D70\\",\\"label\\":\\"Flur-DG-Licht-An\\"},{\\"command\\":\\"tmp6FCB3D-3D3533909\\",\\"label\\":\\"Flur-DG-Licht-Aus\\"},{\\"command\\":\\"tmp6FCB3D-3D3533755\\",\\"label\\":\\"Flur-DG-Licht-Dimmen\\"}]}"}', 'type': 'ChannelDescriptionChangedEvent'} in on_sse_event:
2022-04-08 11:54:08.882 [ERROR] [HABApp                              ] - File "/opt/habapp/lib/python3.7/site-packages/HABApp/openhab/map_events.py", line 39, in get_event
2022-04-08 11:54:08.882 [ERROR] [HABApp                              ] -     27   def get_event(_in_dict: dict) -> OpenhabEvent:
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] -  (...)
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] -     35       payload = load_json(p_str)
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] -     36
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] -     37       # Find event from implemented events
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] -     38       try:
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] - --> 39           return __event_lookup[event_type].from_dict(topic, payload)
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] -     40       except KeyError:
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] -     ..................................................
2022-04-08 11:54:08.883 [ERROR] [HABApp                              ] -      _in_dict = {'topic': 'openhab/channels/avmfritz:fritzbox:192_168_122_1:apply_template/descriptionchanged',
2022-04-08 11:54:08.884 [ERROR] [HABApp                              ] -                  'payload': '{"field":"COMMAND_OPTIONS","channelUID":"avmfritz:fritzbox:192_168_122_1:apply_template","linkedItemNames":["ApplyTemplate"],"value":"{\\"options\\":[{\\"command\\":\\"tmp6FCB3D-3D3532D70\\",\\"label\\":\\"Flur-DG-Licht-An\\"},{\\"command\\":\\"tmp6FCB3D-3D3533909\\",\\"label\\":\\"Flur-DG-Licht-Aus\\"},{\\"command\\":\\"tmp6FCB3D-3D3533755\\",\\"label\\":\\"Flur-DG-Licht-Dimmen\\"}]}"}',
2022-04-08 11:54:08.884 [ERROR] [HABApp                              ] -                  'type': 'ChannelDescriptionChangedEvent'}

**Comment by spacemanspiff2007** [comment]
You can comment out [this line](https://github.com/spacemanspiff2007/HABApp/blob/a123fbfa9928ebb3cda9a84f6984dcba593c8236/src/HABApp/openhab/connection_handler/http_connection.py#L269) in the HABApp file in your local installation to temporarily fix this.

**Comment by spacemanspiff2007** [comment]
Can you verify this with 1.0

**Comment by die-bauerei** [comment]
I can try it out once I updated openhab. I will let you know

**Comment by die-bauerei** [comment]
I can confirm that the problem is fixed in v1.0.4

**Comment by spacemanspiff2007** [comment]
Thanks
